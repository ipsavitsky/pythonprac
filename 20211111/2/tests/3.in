(100, 123123), (234791, -187236), (123787234, 983274932)
