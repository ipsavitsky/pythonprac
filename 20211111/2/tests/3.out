134577666163491.69
